docs/pages/guides/create-app/using-onchain-app-template.mdx

```diff
@@ -0,0 +1,47 @@
+# Getting started with Onchain App Template
+This guide uses [Onchain App Template](https://github.com/coinbase/onchain-app-template/tree/main).
+
+::::steps
+## Create project
+Clone the repository using the command below.
+
+```bash
+git clone https://github.com/coinbase/onchain-app-template.git
+```
+
+## Setup project
+To ensure all components work seamlessly, set the following environment variables in your `.env` file:
+
+- `API KEY`: Find this on the API page at the [Coinbase Dev Portal](https://portal.cdp.coinbase.com/access/api).
+
+- `WALLET_CONNECTOR_PROJECT ID`: Create a project at [Wallet Connect](https://cloud.walletconnect.com) to obtain this.
+
+Add theses to you `.env` file:
+
+```bash
+NEXT_PUBLIC_CDP_API_KEY=ADD_YOUR_API_KEY_HERE
+WALLET_CONNECTOR_PROJECT_ID=ADD_YOUR_PROJECT_ID_HERE
```

Comment:
The env variable is `NEXT_PUBLIC_WC_PROJECT_ID`, please correct this here, and in https://github.com/coinbase/onchain-app-template/blob/main/README.md